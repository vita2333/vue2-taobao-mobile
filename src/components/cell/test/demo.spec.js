import Demo from '../demo'
import { snapshotDemo } from '../../../../tests'

snapshotDemo(Demo)
