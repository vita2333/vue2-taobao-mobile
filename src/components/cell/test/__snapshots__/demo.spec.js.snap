// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders demo correctly 1`] = `
<div>
  <div class="vita-cell">
    <div class="vita-cell__title"><span>新消息提醒</span></div><span class="vita-cell__right-icon"><i class="vita-icon vita-icon--arrow-right"></i></span>
  </div>
  <div class="vita-cell">
    <div class="vita-cell__title"><span>新消息提醒</span></div>
    <div class="vita-cell__value"><span>点击查看</span></div><span class="vita-cell__right-icon"><i class="vita-icon vita-icon--arrow-right"></i></span>
  </div>
  <div class="vita-cell">
    <div class="vita-cell__title"><span>新消息提醒</span>
      <div class="vita-cell__label">关闭后,手机淘宝不运行时将不受到新消息提醒,关闭后,手机淘宝不运行时将不受到新消息提醒</div>
    </div>
    <div class="vita-cell__value"><span>哈哈哈哈</span></div><span class="vita-cell__right-icon"><i class="vita-icon vita-icon--arrow-right"></i></span>
  </div>
  <div class="vita-cell vita-cell--large">
    <div class="vita-cell__title"><span>新消息提醒</span>
      <div class="vita-cell__label">关闭后,手机淘宝不运行时将不受到新消息提醒,关闭后,手机淘宝不运行时将不受到新消息提醒</div>
    </div>
    <div class="vita-cell__value"><span>哈哈哈哈</span></div>
  </div>
  <div class="vita-cell vita-cell--large">
    <div class="vita-cell__title"><span>新消息提醒</span>
      <div class="vita-cell__label">关闭后,手机淘宝不运行时将不受到新消息提醒,关闭后,手机淘宝不运行时将不受到新消息提醒</div>
    </div>
    <div class="vita-cell__value"><span>哈哈</span></div><span class="vita-cell__right-icon"><i class="vita-icon vita-icon--ellipsis"></i></span>
  </div>
</div>
`;
